src/Onboarding/Onboarding.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * Onboarding class
+ *
+ * @package Give
+ */
+
+namespace Give\Onboarding;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Organizes WordPress actions and helper methods for Onboarding.
+ *
+ * @since 2.8.0
+ */
+class Onboarding {
+
+	/**
+	 * Initialize Reports and Pages, register hooks
+	 *
+	 * @since 2.8.0
```

Comment:
Looks like you forgot to update this description, should be Onboarding specific here

Reply:
Ah, missed that one - fixed.